test_enkelt.py

```diff
@@ -1,111 +1,314 @@
 import unittest
-
 import enkelt
 
 
+def get_all_functions_sample_code_file_name(prefix, is_parser):
+    return ''.join(
+        [
+            'TestResources/',
+            prefix,
+            '_sample_code/expected_',
+            'parser' if is_parser else 'lexer',
+            '_output/in_all_functions.txt'
+        ]
+    )
+
+
+def default_get_sample_code(prefix, is_parser):
+    lines = []
+    with open(get_all_functions_sample_code_file_name(prefix, is_parser)) as f:
+        for line in f.readlines():
+            line = line.strip()
+            if line:
+                if is_parser is False:
+                    line = list(eval(line))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of possibly insecure function - consider using safer ast.literal_eval.](https://app.codacy.com/manual/Enkelt/Enkelt-Dev/pullRequest?prid=5205687)